backend/backend-infrastructure/ClassLibrary2/UnitOfWork.cs

```diff
@@ -0,0 +1,33 @@
+﻿using red_team.backend_infrastructure.Repositories;
+using red_team.Repositories.Interfaces;
+using System;
+
+namespace red_team.Repositories
+{
+    public class UnitOfWork : IDisposable
```

Comment:
You also need to create an interface for UnitOfWork. And implement it here. Consumer should use interface, not implementation.